service/scaladsl/client/src/main/scala/com/lightbend/lagom/scaladsl/client/ServiceLocators.scala

```diff
@@ -102,33 +100,33 @@ trait CircuitBreakerComponents {
  * Components for using the configuration service locator.
  */
 trait ConfigurationServiceLocatorComponents extends CircuitBreakerComponents {
-  lazy val serviceLocator: ServiceLocator = new ConfigurationServiceLocator(configuration, circuitBreakersPanel)(executionContext)
+  lazy val serviceLocator: ServiceLocator = new ConfigurationServiceLocator(config, circuitBreakersPanel)(executionContext)
 }
 
 /**
  * A service locator that uses static configuration.
  */
-class ConfigurationServiceLocator(configuration: Config, circuitBreakers: CircuitBreakers)(implicit ec: ExecutionContext)
+class ConfigurationServiceLocator(config: Config, circuitBreakers: CircuitBreakersPanel)(implicit ec: ExecutionContext)
   extends CircuitBreakingServiceLocator(circuitBreakers) {
 
-  @deprecated(message = "prefer constructor using typesafe Config instead", since = "1.4.0")
+  @deprecated(message = "Use constructor accepting {@link com.lightbend.lagom.scaladsl.client.CircuitBreakersPanel} instead", since = "1.4.0")
```

Comment:
Looks like square-bracket links do work in general, but there's no syntax to link to a specific constructor 😦